### PR TITLE
Implement blueprint module

### PR DESCRIPTION
**Pull request by pawloch00**
## Fixes / Features
- add module to generate cluster toolkit blueprint: https://github.com/GoogleCloudPlatform/cluster-toolkit/tree/main
- especially we want to use this blueprint: https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/community/examples/xpk-gke-a3-megagpu.yaml

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Comment by pawloch00** [comment]
cc @IrvingMg  @mbobrovskyi

File: .github/workflows/build_tests.yaml [hunk 1]
```diff
@@ -28,8 +28,24 @@ env:
...
+      cancel-in-progress: false
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+    - name: Install dependencies
+      run : make install-dev
+    - name: Run tests
+      run: pytest src/xpk/
```
Comment (mbobrovskyi) [inline .github/workflows/build_tests.yaml:44]: Can we add `make pytest`?

File: Makefile [hunk 2]
```diff
@@ -11,13 +11,18 @@ PROJECT_DIR := $(realpath $(shell dirname $(firstword $(MAKEFILE_LIST))))
...
+.PHONY: install install_kjob install_kueuectl install_gcloud check_python update-path 
 
 install: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install
 
+install-dev: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install install-pytest 
+
 pip-install:
 	pip install .
 
+install-pytest:
```
Comment (pawloch00) [inline Makefile:23]: It is here

**Review by mbobrovskyi — APPROVED** [review]
Overall LGTM

**Pull request merged**